### ISSUE TITLE
Table data is one off when fetched from template

### ISSUE THREAD

**Issue by pierremonico** [spencermountain/wtf_wikipedia#344]
When doing this: 
```
  async getData() {
    let doc = await wtf.fetch(
      "Template:2019–20_coronavirus_pandemic_data/Switzerland_medical_cases"
    );
    const data = doc.tables(0).json()
  }
```
`data[0]` displays:
```
{
    date: {text: "2020-02-25"},
    ...
    tg: {text: "1"},
    ...
}
```
whereas in the original table the `1` is in canton `ti`:
![image](https://user-images.githubusercontent.com/10991626/77353615-934fbe00-6d41-11ea-9042-9436727e4d74.png)

To put it in context: the first discovered COVID19 case in Switzerland was confirmed in canton Ticino (`ti`) and is displayed as such in the original table, whereas in the fetched data it slides one column to the left to canton Thurgau (`tg`). This is the case for all other rows as well.

Thanks.

**Edit**: I can confirm that this is already the case at `Document.data` level, so it seems unrelated to e.g. `Table.json()` or `Table.keyValue()` methods.

**Comment by spencermountain** [comment]
hey Pierre, i was just looking at this yesterday. it's a really funny template.
I'll try to release it this afternoon.

in the mean-time, you can do this:
```js
wtf.extend((models, templates)=>{
//https://en.wikipedia.org/wiki/Template:Medical_cases_chart
  templates['medical cases chart/row']: (tmpl, list) => {
    let order = [
      'date',
      'deaths_expr',
      'recovery_expr',
      'cases_expr',
      'alt_expr_1',
      '4th_expr',
      '5th_expr',
      'alt_expr_2',
      'col_1',
      'col_1_change',
      'show_col_1',
      'col_2',
      'col_2_change',
      'show_col_2',
      'divisor',
      'numwidth',
      'collabsible',
      'collapsed',
      'id'
    ]
    let obj = parse(tmpl, order)
    list.push(obj)
    return ''
  }
})
```
that should get each row as a seperate template, which should do the trick
cheers

**Comment by spencermountain** [comment]
oh wait, wrong template.
There seems to be a couple of competing formats. I can't keep up.
will check out this one today (too)

**Comment by spencermountain** [comment]
ah, yes this is a rowspan bug. Good eye! thank you.

**Comment by spencermountain** [comment]
hey, i've got a fix for this on dev.
will release it tomorrow

**Comment by spencermountain** [comment]
fixed in `8.1.0`
thanks

**Comment by pierremonico** [comment]
Hi Spencer,

Thanks for the quick fix and for making this.

Stay safe,
Pierre
